app/views/facilities/_quicklinks.html.haml

```diff
@@ -0,0 +1,20 @@
+.quick_links 
+  .add_to_cart
```

Comment:
Looks like the product name got lost.

![screen shot 2018-07-30 at 11 50 21 am](https://user-images.githubusercontent.com/1099111/43411147-fd415194-93ee-11e8-9f50-4cb12fb2c14a.png)

This is also happening on the recent products view on the homepage.